site.go

```diff
@@ -167,10 +240,11 @@ func (s *Site) GetS3Service() (*s3.S3, error) {
 }
 
 func (s *Site) GetPhotoForKey(key string) Renderable {
-	if s.UseImgix {
-		return s.GetImgixPhoto(key)
-	} else {
+	if s.ResizingService == "" {
+		//TODO: maybe warn that you're going raw
```

Comment:
ToDo in code!

Reply:
ah, that was before validation existed, i can drop that now.